llvm/lib/Target/AMDGPU/SIInstructions.td

```diff
@@ -151,6 +151,7 @@ def S_MOV_B64_IMM_PSEUDO : SPseudoInstSI <(outs SReg_64:$sdst),
   let SchedRW = [WriteSALU, Write64Bit];
   let Size = 16; // Needs maximum 2 s_mov_b32 instructions 8 byte long each.
   let Uses = [];
+  let UseNamedOperandTable = 1;
```

Comment:
What is this for?

Reply:
Makes sense, thanks. I did not even know there was a flag for this. I thought it happened automatically.